src/composables/fungibleTokens.ts

```diff
@@ -0,0 +1,288 @@
+import { ref, watch } from 'vue';
+import { useStore } from 'vuex';
+import camelCaseKeysDeep from 'camelcase-keys-deep';
+import BigNumber from 'bignumber.js';
+import { Encoded } from '@aeternity/aepp-sdk';
+import { fetchAllPages, handleUnknownError, toShiftedBigNumber } from '@/utils';
+import {
+  BigNumberPublic,
+  IToken,
+  ITokenList,
+  ITransaction,
+} from '@/types';
+import { PROTOCOL_AETERNITY, TX_DIRECTION } from '@/constants';
+import { aettosToAe, calculateSupplyAmount, categorizeContractCallTxObject } from '@/protocols/aeternity/helpers';
+import FungibleTokenFullInterfaceACI from '@/lib/contracts/FungibleTokenFullInterfaceACI.json';
+import AedexV2PairACI from '@/lib/contracts/AedexV2PairACI.json';
+import ZeitTokenACI from '@/lib/contracts/FungibleTokenFullACI.json';
+import { AE_SYMBOL } from '@/protocols/aeternity/config';
+import { useAccounts } from './accounts';
+import { useAeSdk } from './aeSdk';
+import { useMiddleware } from './middleware';
+import { useTippingContracts } from './tippingContracts';
+import { useNetworks } from './networks';
+
+const availableTokens = ref<ITokenList>({});
+const tokensBalance = ref<Record<string, IToken[]>>({});
+
+export function useFungibleTokens() {
+  const store = useStore();
```

Comment:
`useStore` should not be used out of the component's scope. I suggest to pass the store as the argument the same way as othe composables.